app/controllers/queue_items_controller.rb

```diff
@@ -30,4 +41,13 @@ def new_queue_item_position
   def current_user_queued_video?(video)
     current_user.queue_items.map(&:video).include?(video)
   end
+
+  def update_queue_items
+    ActiveRecord::Base.transaction do
+      params[:queue_items].each do |queue_item_data|
+          queue_item = QueueItem.find(queue_item_data["id"])
```

Comment:
You can use symbols instead of strings as keys for params hash. :) It is hash with indifferent access.